package/CHANGELOG

```diff
@@ -133,6 +133,7 @@ Enhancements
     the capability to allow intermittent behaviour (PR #2256)
 
 Changes
+  * Removes duplicate `NUMBER_TO_ELEMENTS` table (Issue #2699)
```

Comment:
add, "use Z2SYMB instead" or something like that for anyone who might have used it...

Reply:
Fixed in 4019011.